From 1fa50168682a5b83e14eb63df35af981f5ba3440 Mon Sep 17 00:00:00 2001
From: Paul Sajna <sajattack@postmarketos.org>
Date: Wed, 3 Jun 2026 22:23:56 -0700
Subject: [PATCH 128/212] dt-bindings: vendor-prefixes: Add SiliconWorks / LX
 Semicon

https://www.siliconworks.co.kr (from https://github.com/siw-touch)
redirects to https://www.lxsemicon.com/

Devicetree bindings in aforementioned github org use siw prefix

Signed-off-by: Paul Sajna <sajattack@postmarketos.org>
---
 Documentation/devicetree/bindings/vendor-prefixes.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/vendor-prefixes.yaml b/Documentation/devicetree/bindings/vendor-prefixes.yaml
index ba2002969373..ed7a1610090c 100644
--- a/Documentation/devicetree/bindings/vendor-prefixes.yaml
+++ b/Documentation/devicetree/bindings/vendor-prefixes.yaml
@@ -1519,6 +1519,8 @@ patternProperties:
     description: Si-En Technology Ltd.
   "^si-linux,.*":
     description: Silicon Linux Corporation
+  "^siw,.*":
+    description: LX Semicon (formerly SiliconWorks)
   "^sielaff,.*":
     description: Sielaff GmbH & Co.
   "^siemens,.*":
-- 
2.55.0

